jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerDescription.java

```diff
@@ -0,0 +1,25 @@
+package com.google.cloud.tools.jib.api;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+/**
+ * A class containing the representation of the contents of a container. Currently only exposes
+ * "layers", but can be extended to expose {@link ContainerConfiguration} or other informational
```

Comment:
As this comment notes, ContainerConfiguration is left out... it's not in API, but maybe we can put it in API?